### ISSUE TITLE
PEP 541 Request: tapper

### ISSUE THREAD

**Issue by IGalat** [pypi/support#2491]
### Project to be claimed

`tapper`: https://pypi.org/project/tapper

### Your PyPI username

`IGalat`: https://pypi.org/user/IGalat

### Reasons for the request

According to PEP 541, a project is considered abandoned when ALL of the following are met:
- owner not reachable (see Reachability above);
- no releases within the past twelve months; and
- no activity from the owner on the project’s home page (or no home page listed).

To this:

- Yes, I tried finding emails to no success, googling author's github also didn't give results.
- Yes, the old project hasn't been updated since 2015. Also has no description, classifiers, or any other info about the project.
- Yes, there is no homepage.

There are zero dependencies, and statistics show a dead project.

In short, it looks like a throwaway project long abandoned and not really intended for public use in the first place.

I intend to publish a useful FLOSS project on PyPI. 

### Maintenance or replacement?

Replacement

### Source code repositories URLs

Current project's repository page is unknown.

My repository homepage:

https://github.com/IGalat/tapper

### Contact and additional research

I've tried searching for emails, names or any other information that would allow me to contact the author, to no success.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by yeraydiazdiaz** [comment]
Transferred right away as there's very little functionality in the project and no clear trace of the owner..
